scss/devices/brasil/_desktop.scss

```diff
@@ -0,0 +1,153 @@
+@media (min-width: $desktopVersion){
+    // NAV START
+    details {
+        display: none;
+    }
+    nav {
+        height: 10vh;
+        padding: 0 3%;
+        align-items: center;
+        position: sticky;
```

Comment:
z-index:5;
I also added this in home and korea page nav.
Cuz in my page, the pictures in the main is over the nav.
You can add if you need.